### PR TITLE
Try dropping the pre-loading of puppeteer

### PR DESCRIPTION
**Pull request by zozoens31**
This change is [Reviewable](https://reviewable.io/reviews/bayesimpact/docker-react/2675)

### CODE REVIEW

**Review by zozoens31 — COMMENTED** [review]
This might not be needed anymore, since we have pre-installed the packages Chrome needs to run properly

> *[Reviewable](https://reviewable.io/reviews/bayesimpact/docker-react/2675#-:-MNcx3FDEzVn1xY2e8FQ:be1n54y)* status: LGTM missing from one of: pcorpet

**Review by pcorpet — CHANGES_REQUESTED** [review]
> *[Reviewable](https://reviewable.io/reviews/bayesimpact/docker-react/2675)* status: LGTM missing from one of: pcorpet, Unresolved discussions

*[a discussion](https://reviewable.io/reviews/bayesimpact/docker-react/2675#-MNheH3A0LN2Jrz9Za8S:-MNheH3A0LN2Jrz9Za8T:bi5g7uq) (no related file):*
The bug doesn't seem to be fixed, why are you doing that?

---
___
*[Dockerfile, line 16 at r1](https://reviewable.io/reviews/bayesimpact/docker-react/2675#-MNhe4VI3OeYEfx-bRMA:-MNhe4VI3OeYEfx-bRMB:bdbfx7d) ([raw file](https://github.com/bayesimpact/docker-react/blob/d375f62d6309b6b6edce08e65429cf51b2f91394/Dockerfile#L16)):*
> ```Dockerfile
>     libappindicator1 libnss3 lsb-release xdg-utils wget
> 
> # TODO(cyrille): If an ECONRESET error occurs sporadically, get back lines from 6d51ab3.
> ```

Don't keep a TODO if there's nothing todo, just leave that as a comment

**Review by zozoens31 — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/bayesimpact/docker-react/2675)* status: LGTM missing from one of: pcorpet

*[a discussion](https://reviewable.io/reviews/bayesimpact/docker-react/2675#-MNheH3A0LN2Jrz9Za8S:-MNhlxsMEnEl_KIgJRCj:b-omwndw) (no related file):*
<details><summary><i>Previously, pcorpet (Pascal Corpet) wrote…</i></summary><blockquote>

The bug doesn't seem to be fixed, why are you doing that?
</blockquote></details>

Because the bug might be related to missing libs when installing headless chrome from puppeteer

---
___
*[Dockerfile, line 16 at r1](https://reviewable.io/reviews/bayesimpact/docker-react/2675#-MNhe4VI3OeYEfx-bRMA:-MNhm9Us9VNMqqiPeLpq:b-896fix) ([raw file](https://github.com/bayesimpact/docker-react/blob/d375f62d6309b6b6edce08e65429cf51b2f91394/Dockerfile#L16)):*
<details><summary><i>Previously, pcorpet (Pascal Corpet) wrote…</i></summary><blockquote>

Don't keep a TODO if there's nothing todo, just leave that as a comment
</blockquote></details>

Done.

**Review by pcorpet — CHANGES_REQUESTED** [review]
> Reviewed 1 of 1 files at r2.
> *[Reviewable](https://reviewable.io/reviews/bayesimpact/docker-react/2675)* status: LGTM missing from one of: pcorpet

*[a discussion](https://reviewable.io/reviews/bayesimpact/docker-react/2675#-MNheH3A0LN2Jrz9Za8S:-MNiYx-lAt1VS7UtXz_j:bwr5u9j) (no related file):*
<details><summary><i>Previously, zozoens31 (Cyrille Corpet) wrote…</i></summary><blockquote>

Because the bug might be related to missing libs when installing headless chrome from puppeteer
</blockquote></details>

Sorry I don't understand your answer.
It seems that https://github.com/puppeteer/puppeteer/issues/5835  is still open. Can you explain what has changed in our code or in Docker or in pupetteer or in Chrome since we set up that TODO that would hint that it should not happen again please?

---
